test/uk/gov/hmrc/nisp/connectors/IdentityVerificationConnectorSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 HM Revenue & Customs
+ * Copyright 2017 HM Revenue & Customs
```

Comment:
Could you rebase over the latest master changes so these header commits don't appear